src/components/NotionIntegration.tsx

```diff
@@ -470,15 +470,15 @@ export default function NotionIntegration({
           <div className="flex flex-col gap-2">
             <div className="flex items-center gap-2 mb-2">
               <div className="text-sm text-white font-light">
-                Connected to {selectedPageTitle} Notion page
+                Connected to <span className="font-medium underline">{selectedPageTitle}</span> Notion page
               </div>
```

Comment:
for published task, if notion was not connected before, if I click on edit -> connect notion -> do notion auth and then notion redirects back, it opens task in non-edit mode and the notion connection never happens.

<img width="1426" height="654" alt="Screenshot 2025-07-31 at 2 27 50 PM" src="https://github.com/user-attachments/assets/88063c63-ea4f-41ef-8d13-cec5a3729dc7" />